charactersheet/charactersheet/models/common/armor.js

```diff
@@ -25,7 +25,12 @@ function Armor() {
     self.armorStealthOptions = ko.observableArray(Fixtures.armor.armorStealthOptions);
 
     self.acLabel = ko.pureComputed(function() {
-        return 'AC ' + self.armorClass();
+        if (self.armorClass().replace(/^\s+|\s+$/g, '')) {
```

Comment:
Why are we doing this regex? It's looking for whitespace, right? Just curious because we usually don't do this.